runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/FnApiControlClientPoolService.java

```diff
@@ -34,14 +33,13 @@
     implements FnService {
   private static final Logger LOGGER = LoggerFactory.getLogger(FnApiControlClientPoolService.class);
 
-  private final ThrowingConsumer<? super FnApiControlClient> clientPool;
+  private final ControlClientSink clientPool;
```

Comment:
<!--thread_id:cc_182896327_t; commit:1fd121da1417624b3b84f0300648251da64b9cb5; resolved:1-->
<!--section:context-quote-->
> **jkff** wrote:
> Should it be called clientSink, or should the type be ControlClientPool?

<!--section:body-->
It should be called `clientSink`. This is residual from a previous refactor.